gems/activejob/6.0/activejob.rbs

```diff
@@ -21,3 +23,27 @@ module ActiveJob
     def self.discard_on: (*untyped) -> void | ...
   end
 end
+
+module ActiveJob
+  module Logging
+    # nodoc:
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def logger: () -> untyped
+      def logger=: (untyped val) -> untyped
```

Comment:
FYI: The API doc says:

> Accepts a logger conforming to the interface of Log4r or the default Ruby Logger class. You can retrieve this logger by calling logger on either an Active Job job class or an Active Job job instance.
https://api.rubyonrails.org/classes/ActiveJob/Logging.html#method-i-logger

I'm not sure "the interface of Log4r". So I'm okay to keep this untyped.